smarts/env/gymnasium/wrappers/metric/costs.py

```diff
@@ -484,7 +494,7 @@ class CostFuncsBase:
     # fmt: off
     collisions: Callable[[], Callable[[RoadMap, VehicleIndex, Done, Observation], Costs]] = _collisions
     comfort: Callable[[], Callable[[RoadMap, VehicleIndex, Done, Observation], Costs]] = _comfort
-    dist_to_destination: Callable[[Point,float], Callable[[RoadMap, VehicleIndex, Done, Observation], Costs]] = _dist_to_destination
+    dist_to_destination: Callable[[str, float, Point], Callable[[RoadMap, VehicleIndex, Done, Observation], Costs]] = _dist_to_destination
```

Comment:
Do we have a description of what these metrics are intended to do (even if the implementation is private?)

Reply:
Although not available currently, a description of the metrics is on the list of things to do next. It is tracked in issue #1741.